packages/dataviews/src/dataforms-layouts/panel/index.tsx

```diff
@@ -111,7 +112,7 @@ function FormField< Item >( {
 							) }
 							onClick={ onToggle }
 						>
-							<field.render item={ data } />
+							<field.render item={ data } view={ LAYOUT_PANEL } />
```

Comment:
How this looks like in the regular panel? Can we update the storybook to see how fields of this type behave?